code/modules/reagents/oldchem/chemical_reaction/chemical_reaction_misc.dm

```diff
@@ -105,6 +105,13 @@
 		result_amount = 3
 		mix_message = "Ick, this stuff really stinks. Sure does make the container sparkle though!"
 
+	luminol
+		name = "Luminol"
+		id = "luminol"
+		result = "luminol"
+		required_reagents = list("hydrogen" = 2, "carbon" = 2, "ammonia" = 2)
```

Comment:
this will probably cause problems with other recipes.

I'd recommen: 
welding fuel+ammonia+space cleaner+sodium+sulfur